src/utils/composables.js

```diff
@@ -67,7 +67,7 @@ export function useIntegerRouteParam (name) {
   const route = useRoute()
   return computed({
     get () {
-      return route.params[name] && parseInt(route.params[name], 10)
+      return route?.params[name] && parseInt(route.params[name], 10)
```

Comment:
I would prefer to always have a router, which might be a fake one (from `import '>/mockRouter'`) or a real one... but I think this might conceal bugs rather than reveal them. Hard to notice if we accidentally don't have a router when we are expecting to. WDYT?

Reply:
Agree, don't really like this solution either. But as far as I understood, `vue-router-mock` is somehow based on vue-test-utils, so I wonder how it works in storybook. `>/routerMocks` requires jest, so that wouldn't work.

Reply:
I undid this change, accepting that storybook won't work (for now).